frontend/pages/team/index.tsx

```diff
@@ -0,0 +1,78 @@
+'use client';
+
+import React , { useState } from 'react'
+
+
+import './../../styles/team.module.css'
+import Card from '@/components/Card';
+// import MemberList from './MemberList';
+
+const OurTeam = () => {
+    const styleObj = {
+        background: ";",
```

Comment:
what is the use of this line

Reply:
line 12???
no use ...
resolved by give tailwnd classes